### ISSUE TITLE
Purestake API v1 is deprecated

### ISSUE THREAD

**Issue by gianlucamazza** [algoduino/algoduino#1]
https://www.purestake.com/blog/algorand-api-and-indexer-v2/

**Comment by TedNIVAN** [comment]
Yes, we are aware of that. We are currently reshaping the project.
I will let you know where we're at asap.
